packages/next-codemod/bin/upgrade.ts

```diff
@@ -86,10 +86,14 @@ export async function runUpgrade(
     peerDependencies: Record<string, string>
   }
 
-  const res = await fetch(`https://registry.npmjs.org/next/${revision}`)
-  if (res.status === 200) {
-    targetNextPackageJson = await res.json()
-  }
+  try {
+    const targetNextPackage = execSync(
+      `npm --silent view "next@${revision}" --json`,
+      { encoding: 'utf-8' }
+    )
+    targetNextPackageJson = JSON.parse(targetNextPackage)
+  } catch {}
```

Comment:
```suggestion
  } catch (err) {
```
Not sure we should fully suppress the error here but it was before so can be looked at in follow-up